network/hybridNetwork.go

```diff
@@ -48,7 +47,14 @@ func NewHybridP2PNetwork(log logging.Logger, cfg config.Local, datadir string, p
 	p2pcfg.NetAddress = cfg.P2PHybridNetAddress
 	p2pcfg.IncomingConnectionsLimit = cfg.P2PHybridIncomingConnectionsLimit
 	identityTracker := NewIdentityTracker()
-	p2pnet, err := NewP2PNetwork(log, p2pcfg, datadir, phonebookAddresses, genesisID, networkID, nodeInfo, &identityOpts{tracker: identityTracker})
+
+	var p2pDummyMeshCreator *dummyMeshCreator
```

Comment:
Question for reviewers: another option is to make `HybridRelayMeshStrategyCreator` "smarter" and take `net` as parameter and move this params saving logic inside it by conditioning on type asserting.

Reply:
I first read this and thought it was test code based solely on the name!